package.json

```diff
@@ -18,26 +18,27 @@
         "node": ">=10.15.0"
     },
     "dependencies": {
-        "axios": "0.24.0",
-        "express": "4.17.1",
-        "express-session": "1.17.2",
+        "axios": "^0.26.1",
+        "connect-pg-simple": "^7.0.0",
+        "express": "^4.17.3",
+        "express-session": "^1.17.2",
         "jade": "1.11.0",
```

Comment:
This dependency is now giving critical security warnings, but I think it is actually not needed - in app.js we have 

    app.set("view engine", "jade");

but I think we don't need a view engine since our application renders no HTML UI.